tests/spec/fs.stat.spec.js

```diff
@@ -92,3 +92,24 @@ describe('fs.stat', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+
+describe('fsPromises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should return an error if path does not exist', function(done) {
```

Comment:
We don't use `function(done)` for promises. You should use `function()` instead.